clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -100,8 +100,6 @@
  * {@code
  * Properties props = new Properties();
  * props.put("bootstrap.servers", "localhost:9092");
- * props.put("acks", "all");
- * props.put("retries", 0);
```

Comment:
In V3.0, `acks` is default to `all`, and `retires` is not recommended to set to 0 after KIP-91(https://cwiki.apache.org/confluence/display/KAFKA/KIP-91+Provide+Intuitive+User+Timeouts+in+The+Producer)